ios/PolyPodApp/Core/UpdateNotification.swift

```diff
@@ -107,74 +89,67 @@ class UpdateNotification {
     }
     
     private static let notificationData = Data.load()
-        
-    private static func loadLastState(_ id: Int) -> State {
-        if id == 0 {
-            return .ALL_SEEN
-        }
-        
-        guard let (lastId, lastState) = LastUpdateNotification.read() else {
-            return .NOT_SEEN
-        }
-        
-        if id < lastId {
-            return .ALL_SEEN
-        }
-        
-        if id != lastId {
-            return .NOT_SEEN
-        }
-        
-        return State.parse(lastState) ?? .ALL_SEEN
-    }
     
-    let id = UpdateNotificationMockId.read() ?? notificationData.id
     let pushDelay = notificationData.pushDelay
     let title = notificationData.localizedTitle
     let text = notificationData.localizedText
     
-    private var cachedState: State
-    private var state: State {
-        get { cachedState }
-        set {
-            cachedState = newValue
-            LastUpdateNotification.write(id: id, state: cachedState.rawValue)
-        }
-    }
-    
+    private let raw: OpaquePointer
```

Comment:
This is the pointer to UpdateNotification from core. OpaquePointer is used to described incomplete types, as it is the case for UpdateNotification.